enhancements/installer/credentials-management-outside-openshift-cluster.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Credentials Management outside OpenShift Cluster
+authors:
+  - "@akhil-rane"
+reviewers:
+  - "@abhinavdahiya"
+  - "@dgoodwin"
+  - "@joelddiaz"
+approvers:
+  - "@derekwaynecarr"
+  - "@sdodson"
+creation-date: 2021-03-25
+last-updated: 2021-03-25
+status: provisional
+---
+
+# Credentials Management outside OpenShift Cluster
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The intent of this enhancement is to take the process of credentials management outside the OpenShift cluster for new 
+platforms. We propose to make *manual* mode as default for clusters on new platforms. 
+
+## Motivation
+
+The main motivation behind this enhancement is to satisfy the customer requirement to follow the best security practices 
+by not storing admin credentials inside the cluster
+
+### Goals
+
+As part of this enhancement we plan to do the following:
+* Set *manual* as default/preferred credentials mode for OpenShift
+* Not store admin level cloud credentials inside a cluster
+* Provide a tooling design for credentials management outside the cluster
```

Comment:
Might need to tighten these up a little, I would restate the goals as:

  * Guide new platforms to start with Manual mode rather than Mint mode.
  * Provide optional tooling (ccoctl) for administrators to manage their credentials outside the cluster. (prep for install, reconcile, prep for upgrade)
  * Establish precedent for future direction of prefering Manual over Mint.

Reply:
100% agree with above.